ocean_lib/data_provider/data_service_provider.py

```diff
@@ -105,6 +106,37 @@ def encrypt_files_dict(
 
         return ""
 
+    @staticmethod
+    @enforce_types
+    def encrypt_urls(
+        files: list,
+        encrypt_endpoint: str,
+    ) -> str:
+        data = str(list(map(lambda file: file.from_dict(), files)))
+        response = DataServiceProvider._http_method(
+            "post",
+            encrypt_endpoint,
+            data=data,
+            headers={"content-type": "application/octet-stream"},
+        )
+        if response and hasattr(response, "status_code"):
```

Comment:
please remove the multiple if imbrication by using my favourite tactic :D

Reply:
??? the other way around! This will return "" if there is a status code. It will never reach the next if!

Reply:
I don't know how this was resolved. And I don't know how it is working.